src/main/scala/io/sdkman/changelogs/AntMigration.scala

```diff
@@ -0,0 +1,203 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "030")
+class AntMigration {
+  @ChangeSet(
+    order = "001",
+    id = "001-add_ant_1.9.10",
+    author = "andrebrait"
+  )
+  def migration001(implicit db: MongoDatabase): Unit = {
+    Version(
+      "ant",
+      "1.9.10",
+      "https://archive.apache.org/dist/ant/binaries/apache-ant-1.9.10-bin.zip"
+    ).validate()
+      .insert()
+      .asCandidateDefault()
```

Comment:
This only needs to be set once on the version that should be marked as default. How about only putting it on `1.10.8`?